package.json

```diff
@@ -1,23 +1,28 @@
 {
     "name": "@bazel/typescript",
     "description": "Build TypeScript with Bazel",
-    "version": "0.5.0",
-    "keywords": ["typescript", "bazel"],
+    "version": "0.5.3",
```

Comment:
Is this related to this change?

Reply:
no, can be separate also. Sorry this was so messy.